<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleWindow</class>
 <widget class="QMainWindow" name="ScheduleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="MD">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ПН</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ВТ</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>СР</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ЧТ</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ПТ</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>СБ</string>
    </property>
   </widget>
   <widget class="QLabel" name="MD_7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ВС</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>931</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад в Меню</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>480</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1455</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCloseMen">
   <property name="text">
    <string>CloseMen</string>
   </property>
  </action>
  <action name="switchToMainAction">
   <property name="text">
    <string>Меню</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
